### ISSUE TITLE
SqlAlchemy serializer. Nuberic float field error.

### ISSUE THREAD

**Issue by fantom0005** [Relrin/aiorest-ws#82]
Hello, my friend)
An error occurred during the serialization of float fields.
Error:
```bash
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/asyncio/coroutines.py", line 210, in coro
    res = func(*args, **kw)
  File "/Users/antonsvecov/envs/wl/lib/python3.6/site-packages/aiorest_ws/request.py", line 63, in onMessage
    response = self.factory.router.process_request(request)
  File "/Users/antonsvecov/envs/wl/lib/python3.6/site-packages/aiorest_ws/routers.py", line 131, in process_request
    response.content = handler.dispatch(request, *args, **kwargs)
  File "/Users/antonsvecov/envs/wl/lib/python3.6/site-packages/aiorest_ws/views.py", line 77, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/antonsvecov/Projects/betronic-white-label/admin/src/utils/decorators.py", line 21, in wrapper
    return handler(request, *args, **kwargs)
  File "/Users/antonsvecov/Projects/betronic-white-label/admin/src/utils/abstract_view.py", line 30, in get
    session, **kwargs)
  File "/Users/antonsvecov/Projects/betronic-white-label/admin/src/utils/abstract_view.py", line 20, in _get_model_all
    return self.serializer(items, many=True).data
  File "/Users/antonsvecov/envs/wl/lib/python3.6/site-packages/aiorest_ws/db/orm/serializers.py", line 635, in data
    ret = super(ListSerializer, self).data
  File "/Users/antonsvecov/envs/wl/lib/python3.6/site-packages/aiorest_ws/db/orm/serializers.py", line 154, in data
    self._data = self.to_representation(self.instance)
  File "/Users/antonsvecov/envs/wl/lib/python3.6/site-packages/aiorest_ws/db/orm/serializers.py", line 585, in to_representation
    return [self.child.to_representation(item) for item in data]
  File "/Users/antonsvecov/envs/wl/lib/python3.6/site-packages/aiorest_ws/db/orm/serializers.py", line 585, in <listcomp>
    return [self.child.to_representation(item) for item in data]
  File "/Users/antonsvecov/envs/wl/lib/python3.6/site-packages/aiorest_ws/db/orm/serializers.py", line 379, in to_representation
    fields = self._readable_fields
  File "/Users/antonsvecov/envs/wl/lib/python3.6/site-packages/aiorest_ws/utils/functional.py", line 17, in __get__
    result = instance.__dict__[self.func.__name__] = self.func(instance)
  File "/Users/antonsvecov/envs/wl/lib/python3.6/site-packages/aiorest_ws/db/orm/serializers.py", line 291, in _readable_fields
    field for field in self.fields.values()
  File "/Users/antonsvecov/envs/wl/lib/python3.6/site-packages/aiorest_ws/db/orm/serializers.py", line 277, in fields
    for key, value in self.get_fields().items():
  File "/Users/antonsvecov/envs/wl/lib/python3.6/site-packages/aiorest_ws/db/orm/serializers.py", line 756, in get_fields
    fields[field_name] = field_class(**field_kwargs)
TypeError: __init__() missing 2 required positional arguments: 'max_digits' and 'decimal_places'
```

Model:
```python
class UserModel(BaseModel):
    __tablename__ = "user"

    id = Column(Integer, autoincrement=True, primary_key=True)
    first_name = Column(String(60))
    middle_name = Column(String)
    last_name = Column(String(60))
    nickname = Column(String(60), unique=True)
    balance = Column(Numeric(10, 2), default=0)
```

Problem in the field balance.
Please, fix this bug.
Thanks, dear friend)

**Comment by Relrin** [comment]
Hi, @fantom0005!

Thanks for opening an issue here.
This error occurred in runtime because I forgot to add additional logic for processing cases if passed [Numeric field](http://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.Numeric) (or some derived class), like we have it for Django DecimalField type.

**Comment by Relrin** [comment]
Fixed in [pull request #83](https://github.com/Relrin/aiorest-ws/pull/83)
